internal/cli/atlas/quickstart/dbuser_setup.go

```diff
@@ -97,3 +96,13 @@ func (opts *Opts) newDatabaseUser() *atlas.DatabaseUser {
 		Username:     opts.DBUsername,
 	}
 }
+
+func GeneratePassword() (string, error) {
```

Comment:
[nit] let's either make it private or add tests

Reply:
done! Made it private